samples/as_json.erb

```diff
@@ -0,0 +1,3 @@
+<% remote_resource.as_json('https://my-json-server.typicode.com/typicode/demo/posts', []).each do |el| %>
```

Comment:
All the example are ready to use examples, I propose to use a real URL.

Example to display all the version of consul-templaterb with `samples/consul-templaterb-versions.html`:
```
<% remote_resource.as_json('https://rubygems.org/api/v1/versions/consul-templaterb.json', [], refresh_delay_secs: 1800).each do |gem_version| %>
* Version: <%= gem_version['number'] %> (<%= gem_version['created_at'] %>) with <%= gem_version['downloads_count'] %> downloads
<% end %>
```

(Of course, you have to update your mock to make it work)

Reply:
good idea, thanks!

Reply:
done